scopes/harmony/aspect-loader/aspect-definition.ts

```diff
@@ -45,6 +47,22 @@ export class AspectDefinition {
     return null;
   }
 
+  get packageName() {
+    return this.getId ? getCoreAspectPackageName(this.getId) : null;
```

Comment:
What if it's not a core aspect but user aspect?
This will return wrong result

Reply:
I see. I updated the implementation. I didn't reuse `workspace.componentPackageName()` because there is no workspace instance to consume. Please let me know if there are any better ways to do that. 🙏